### DISCUSSION TITLE
formily引入问题

### DISCUSSION THREAD

**Discussion by nzhang3** [alibaba/formily#3739] [Q&A 1.x]
我想在单个页面用script标签引入formily总是报错，cdn.jsdelivr.net/npm/@f，已经提前引入了vue3,formily.core等文件，引入这个formily.vue的时候报错
Uncaught TypeError: window.Vue.use is not a function

**Comment by zxq08** [comment]
嗨 请问下这个问题有后续吗  我也在考虑把formily放到cdn，前来学习
